pkg/tfvars/aws/aws.go

```diff
@@ -1,44 +1,26 @@
 package aws
 
-// Endpoints is the type of the AWS endpoints.
-type Endpoints string
-
 const (
-	// EndpointsAll represents the configuration for using both private and public endpoints.
-	EndpointsAll Endpoints = "all"
-	// EndpointsPrivate represents the configuration for using only private endpoints.
-	EndpointsPrivate Endpoints = "private"
-	// EndpointsPublic represents the configuration for using only public endpoints.
-	EndpointsPublic Endpoints = "public"
+	// DefaultVPCCIDRBlock is the default CIDR range for an AWS VPC.
+	DefaultVPCCIDRBlock = "10.0.0.0/16"
+	// DefaultRegion is the default AWS region for the cluster.
+	DefaultRegion = "us-east-1"
 )
 
 // AWS converts AWS related config.
 type AWS struct {
-	EC2AMIOverride string    `json:"aws_ec2_ami_override,omitempty"`
-	Endpoints      Endpoints `json:"aws_endpoints,omitempty"`
-	External       `json:",inline"`
+	EC2AMIOverride string            `json:"aws_ec2_ami_override,omitempty"`
 	ExtraTags      map[string]string `json:"aws_extra_tags,omitempty"`
-	InstallerRole  string            `json:"aws_installer_role,omitempty"`
 	Master         `json:",inline"`
 	Region         string `json:"aws_region,omitempty"`
-	VPCCIDRBlock   string `json:"aws_vpc_cidr_block,omitempty"`
+	VPCCIDRBlock   string `json:"aws_vpc_cidr_block"`
 	Worker         `json:",inline"`
 }
 
-// External converts external related config.
-type External struct {
-	MasterSubnetIDs []string `json:"aws_external_master_subnet_ids,omitempty"`
-	PrivateZone     string   `json:"aws_external_private_zone,omitempty"`
-	VPCID           string   `json:"aws_external_vpc_id,omitempty"`
-	WorkerSubnetIDs []string `json:"aws_external_worker_subnet_ids,omitempty"`
-}
-
 // Master converts master related config.
 type Master struct {
-	CustomSubnets    map[string]string `json:"aws_master_custom_subnets,omitempty"`
-	EC2Type          string            `json:"aws_master_ec2_type,omitempty"`
-	ExtraSGIDs       []string          `json:"aws_master_extra_sg_ids,omitempty"`
-	IAMRoleName      string            `json:"aws_master_iam_role_name,omitempty"`
+	EC2Type          string `json:"aws_master_ec2_type,omitempty"`
+	IAMRoleName      string `json:"aws_master_iam_role_name,omitempty"`
```

Comment:
The commit message for 4f85bc28a8 claims to be dropping this, but you aren't actually dropping it in the diff.  I don't really care one way or the other (we can always remove it in follow-up work if you want to punt here), but the commit message and diff should agree on whether it's staying or going ;).